### PR TITLE
(gray did 99% of the work) Removes all tasers except for AI turrets, leaving the HoS gun with a weak variant. 

### PR DESCRIPTION
**Pull request by silicons**
## About The Pull Request

#10131 but modified to give the HoS gun a slowdown only taser rather than one hit combat mode disable (it'll do a bit more stamina though) and AI turrets having oldtasers if they didn't already

oh and im going to make the hos gun switch to disabler automatically when firing the taser :^)

## Why It's Good For The Game

Multiple taser reworks have failed, as much as I like having options this hasn't worked and this really isn't going to work at this point.

## Changelog 
:cl: GrayRachnid, kevinz000
del: Tasers have been for the most part removed, other than the HoS's 
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/energy/energy_gun.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 /obj/item/gun/energy/e_gun/mini
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/energy/energy_gun.dm:13]: Can we give this to sec rather then normal disablers? They are rather under used

**Comment by Poojawa** [comment]
Hey remember that time you threatened to ban me because I copy/pasted the job loadouts which gave them disablers instead of tasers. 

And you got really upset because you thought it was a deliberate change, and then even more so when I was entertaining the idea in OOC chat to shake things up?

hmm.

**Comment by silicons** [comment]
ah yes the difference between implying an unintentional change was an intentional undocumented change during a refactor and an intentional pr doing said change
k

**Comment by Poojawa** [comment]
man 'undocumented change' is your favorite phrase of 2020 so far. lol

**Comment by Ghommie** [comment]
#10762 already dealt with this balance issue.

**Pull request closed without merging**